Programming/bundle/nlparser/src/main/webapp/jsp/tabs.jsp

```diff
@@ -34,6 +34,7 @@
 
                     <input id="search-input-${domainEntry.key}" name="q" type="text" class="input-large search-input"  
                            placeholder="Type your question.." data-searchdomain="${domainEntry.key}"></input>
+                    <input type="submit" style="position: absolute; left: -9999px"/>
```

Comment:
Yes it works, but if I switch to using display:none, in IE Enter no longer submits the form. I think the way IE handles display:none is removing the element completely from the tree.